udata/core/dataset/search.py

```diff
@@ -83,7 +83,8 @@ class Meta:
     resources = Object(properties={
```

Comment:
Commenting for the whole file.

I'm not sure to understand how this class works. I've tried to look at various fields and added the necessary lines. 
I don't understand if I need to add (in order of changes, top to bottom):
- a property
- a facet
- update the `serialize` method

Reply:
I'm not sure either :-/ let's keep this for a future PR.